src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -120,6 +120,11 @@ def disable_attention_slicing(self):
         self.enable_attention_slicing(None)
 
     def enable_sequential_cpu_offload(self):
+        r"""
+        Offloads all models to CPU using accelerate, drastically reducing memory usage. When called, unet,
```

Comment:
```suggestion
        Offloads all models to CPU using accelerate, significantly reducing memory usage. When called, unet,
```
Maybe a bit less extreme

Reply:
Yeah, much better.